simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/mock/OcppServerClient.java

```diff
@@ -0,0 +1,47 @@
+package com.evbox.everon.ocpp.simulator.mock;
+
+import io.undertow.websockets.core.WebSocketChannel;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * A client that holds websocket senders.
+ *
+ * A websocket sender can be retrieved by station_id.
+ */
+public class OcppServerClient {
```

Comment:
IMO having a client is too much. We can use mock directly

Reply:
You can, but again Single-responsibility. Dont try to put everything in one bucket. Look at spring-test project. Every class has its own purpose. MockServer only mocks, it should not have a send functionality.

Reply:
We're not making a general-purpose widely used class. It's a tool for specific use-case, I don't see a problem with combining logic

Reply:
How about extendable, readable, scalable code? We should follow patterns and best-practises for that...